ui-particles-angular/projects/ui-particles-angular/src/lib/gio-submenu/gio-submenu-subitem/gio-submenu-subitem.component.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+@use '../../../scss' as gio;
+
+$activeBackground: map.get(gio.$mat-space-palette, lighter30);
+$text: map.get(gio.$mat-decorative-palette, lighter-background);
+$textLight: map.get(gio.$mat-content-palette, lighter);
+
+:host {
+  display: block;
+}
+
+.gio-submenu-subitem {
+  width: 234px;
+  height: 52px;
+  flex: none;
+  flex-grow: 0;
+  order: 2;
+  padding: 16px 32px;
+  border-radius: 4px;
+  cursor: pointer;
+
+  &__active {
+    background: $activeBackground;
+  }
+
+  &__title {
+    width: 37px;
+    height: 20px;
+    flex: none;
+    flex-grow: 0;
+    order: 0;
+    color: #fff;
```

Comment:
Same for all text color
I would have expected to see a contrast color of the item's background 🤔 WDYT ?